### ISSUE TITLE
Move Perspective->Align features to Transform menu

### ISSUE THREAD

**Issue by pseidemann** [godotengine/godot-proposals#5917]
### Describe the project you are working on

any

### Describe the problem or limitation you are having in your project

currently, the "Perspective" menu inside the editor viewport has two features/options which stick out:

- Align Transform with View
- Align Rotation with View

I see three problems with that:

1. there is already a "Transform" menu for changing the transform of a node but these two options mentioned are not there to find
2. the "Perspective" menu is for changing the view of the editor viewport. these options however mutate nodes. I think it can be assumed that a user is not expecting these options to be present in this menu
3. these options are way too buried inside this menu for the very common task of aligning a Camera node with the current editor's view. I use godot for a couple of month now and never knew these options exist and never saw one using this feature in youtube tutorials when aligning a Camera node. see also #5916

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

move the two options
- Align Transform with View
- Align Rotation with View

to either the "Transform" menu in the action bar (not sure how it is called exactly, I'm talking about the horizontal buttons bar right below the open scenes tabs)
\- or -
add them as dedicated buttons to the action bar

this will make the "Perspective" menu only changing perspectives (no mutate actions on nodes) and these transform actions are next to the existing transforming actions (e.g. "snap object to floor")

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

moving position of buttons (see feature description)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

unknown

### Is there a reason why this should be core and not an add-on in the asset library?

movement of existing buttons of core functionality

**Comment by Calinou** [comment]
Remember that you can split the 3D viewport into more than one view (up to 4).

One issue is that the Transform menu is global for all viewports, while the Perspective menu is scoped to each viewport. Aligning selection to view is an action that is specific to a viewport, not one that works the same regardless of which viewport you run it in.

**Comment by pseidemann** [comment]
good point! afaict there is always an active/focused viewport (I see a gray border around the last "touched" viewport) so this one could be used when triggering the transform.

however, I think it would make sense to make this "active" viewport more visible, if there are multiple viewports, e.g. with a 2px red border instead of a 1px gray border, which currently is almost invisible.

**Comment by Calinou** [comment]
> however, I think it would make sense to make this "active" viewport more visible, if there are multiple viewports, e.g. with a 2px red border instead of a 1px gray border, which currently is almost invisible.

The active viewport border used to be more visible, but it was made more subtle due to popular demand: https://github.com/godotengine/godot/pull/48567

**Comment by pseidemann** [comment]
if we add the new menu "Align" to the right side of the "Perspective" menu, we could make more room for it by renaming "Perspective" to "View" (after all it's about viewing and the VIEWport) and rename "View" in the action bar to "Layout" (since it's changing the layout of the "views").

separate comment to give 👍 or 👎

**Comment by timothyqiu** [comment]
> if we add the new menu "Align" to the right side of the "Perspective" menu, we could make more room for it by renaming "Perspective" to "View"

"Perspective" is not the name of that menu, but the current viewport mode. It could also be Orthogonal, Left Perspective, Front Orthogonal, etc.

**Comment by romgerman** [comment]
Accessing the "Align Transform..." items from the bottom of the Viewport options is very unintuitive. You would assume the "Transform" menu will have the options to... *transform* your nodes.

**Comment by timothyqiu** [comment]
> You would assume the "Transform" menu will have the options to... _transform_ your nodes.

Yes, these Align options transform the selected nodes.
